import_export/resources.py

```diff
@@ -1118,6 +1116,19 @@ def _get_ordered_field_names(self, order_field):
         [order.append(f) for f in defined_fields if f not in order]
         return tuple(order) + tuple(k for k in self.fields if k not in order)
 
+    def _check_import_id_fields(self, headers):
+        import_id_fields = [self.fields[f] for f in self.get_import_id_fields()]
+        for field in import_id_fields:
```

Comment:
Do you think it would be better to use a list comprehension here and then error for all the import id fields that are missing?  Might be more performant at a very small level too lol.  

with the OR statement it might be a little clunky to be expressed as a comprehension, but I suspect doable.  

the main thing is, functionally, it would be useful to know all of the missing import id fields instead of just the first one that's missing.

Reply:
Good suggestion - have added this.